bokehjs/src/coffee/renderer/glyph/patches.coffee

```diff
@@ -31,6 +71,10 @@ class PatchesView extends Glyph.View
     return (x[4].i for x in @index.search([x0, y0, x1, y1]))
 
   _render: (ctx, indices, {sxs, sys}) ->
+    # @sxss and @syss are used by _hit_point and sxc, syc
```

Comment:
Let's go ahead and merge it as is... re-reading, I am not against that anymore...